circuit_puzzles/oracle.clsp

```diff
@@ -28,20 +29,31 @@
     ; recreate yourself to conserve updates so far
     (verify-announcement-asserts
       input_announcement_asserts
-      (list
-        (list CREATE_COIN
-          (curry_hashes MOD_HASH
-            (sha256 ONE MOD_HASH)
-            (sha256tree STATUTES_STRUCT)
-            (sha256 ONE ATOM_ANNOUNCER_PUZZLE_HASH)
-            (sha256tree PRICES)
+      (assign
+        (current_timestamp price_expiration_seconds) announce_args
```

Comment:
price_expiration_seconds not asserted

Reply:
it is asserted with the message by statutes